src/main/java/geometry/Square.java

```diff
@@ -0,0 +1,15 @@
+package geometry;
+
+public class Square extends Rectangular {
+
+    public Square(double sideA) {
+        // add default sideB = 0, because we really do not need sideB for Square
+        super(sideA, 0);
```

Comment:
you can simply add double sideA variable to this class, and assign it in the constructor (and do not use super - it means that you will "override" it by doing it)

Reply:
1. I really do not understand what I should to do according to your comment. Can you, please wide the explanation?
2. Is my implementation work badly and do not meets the AC?

Reply:
1. Square class should have only one parameter - e.g. sideA and its own constructor that doesn't have super, because we do not need to reuse parent constructor and override getSquare() method inside Square class. We should use super for all classes that have the same constructor as parent class. It may look the next way (also, you need to add empty constructor to the parent class - "public Rectangular() { }").
<img width="636" alt="image" src="https://user-images.githubusercontent.com/90613721/147113033-3a4a5bb7-9828-4509-88ef-3ec0ec216cce.png">
2. Your implementation works fine, but we try to understand the main OOP principles and follow them (according to the SOLID principle as well).

Reply:
done